### PR TITLE
metrics: add loading region cache duration

### PR DESCRIPTION
**Pull request by you06**
Signed-off-by: you06 <you1474600@gmail.com>

### What problem does this PR solve?

Issue Number: ref #34106

Problem Summary:

tikv/client-go#502 collects the duration of loading region cache and this PR add it to the dashboard.

### What is changed and how it works?

image

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
Add loading region cache duration metrics.
```

### CODE REVIEW

**Comment by sticnarf** [comment]
I'm not sure if these panels should be put in the "Meta" row. To my understanding "meta" refers to some internal data used by TiDB while region cache is not.

The "KV Request" row may be more appropriate?

File: metrics/grafana/tidb.json [hunk 1]
```diff
@@ -12410,6 +12410,93 @@
...
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
           "targets": [
             {
               "expr": "sum(rate(tidb_tikvclient_region_cache_operations_total{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", result=\"err\"}[1m])) by (type)",
```
Comment (cfzjywxk) [inline metrics/grafana/tidb.json:12502]: How about adding an error suffix like `{{type}}-err` for errors.

**Comment by you06** [comment]
> I'm not sure if these panels should be put in the "Meta" row.

Because "Region Cache Error OPS" panel is put in "Meta" now, so I just put the related panels together.

> The "KV Request" row may be more appropriate?

Agree, will move all region cache panels into "KV Request" row.

File: metrics/grafana/tidb.json (hunk 1 above)
Comment (you06) [reply metrics/grafana/tidb.json]: Good idea.

**Comment by cfzjywxk** [comment]
@sticnarf 
PTAL

**Review by sticnarf — APPROVED** [review]

**Pull request merged**